vng_api_common/management/commands/patch_error_contenttypes.py

```diff
@@ -19,6 +19,15 @@ def add_arguments(self, parser):
             "api-spec", help="Path to the openapi spec. Will be overwritten!"
         )
 
+    def patch_response(self, response):
+        content = {}
+
+        for contenttype, _response in response["content"].items():
+            contenttype = ERROR_CONTENT_TYPE
+            content[contenttype] = _response
```

Comment:
shouldn't this also delete the original/incorrect content-type key/value pair? Though it's weird that the previous version of this code also didn't do that.